### ISSUE TITLE
[Retail Bug Report] 3x ...ceElvUI/Mainline/Modules/Skins/GuildBank.lua:35: attempt to index field 'GuildBankTransactionsScrollFrame' (a nil value)

### ISSUE THREAD

**Issue by jeremy-hill** [tukui-org/ElvUI#959]
### What is the issue you are having?

This error is thrown whenever I click the guild bank:
```
Message: ...ce/AddOns/ElvUI/Mainline/Modules/Skins/GuildBank.lua:35: attempt to index field 'GuildBankTransactionsScrollFrame' (a nil value)
Time: Wed May  3 16:45:04 2023
Count: 1
Stack: ...ce/AddOns/ElvUI/Mainline/Modules/Skins/GuildBank.lua:35: attempt to index field 'GuildBankTransactionsScrollFrame' (a nil value)
[string "=(tail call)"]: ?
[string "@Interface/AddOns/ElvUI/Mainline/Modules/Skins/GuildBank.lua"]:35: in function <...ce/AddOns/ElvUI/Mainline/Modules/Skins/GuildBank.lua:16>
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua"]:1856: in function `CallLoadedAddon'
[string "@Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua"]:1797: in function `?'
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:109: in function <...ies/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:19: in function <...ies/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceEvent-3.0/AceEvent-3.0.lua"]:120: in function <...UI_Libraries/Core/Ace3/AceEvent-3.0/AceEvent-3.0.lua:119>
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface/FrameXML/UIParent.lua"]:545: in function `UIParentLoadAddOn'
[string "@Interface/FrameXML/UIParent.lua"]:608: in function `loadFunc'
[string "@Interface/FrameXML/PlayerInteractionFrameManager.lua"]:179: in function `ShowFrame'
[string "@Interface/FrameXML/PlayerInteractionFrameManager.lua"]:215: in function <Interface/FrameXML/PlayerInteractionFrameManager.lua:212>

Locals: (*temporary) = <function> defined =[C]:-1

```

Also, I'm not sure the guild bank is being skinned right. 
![image](https://user-images.githubusercontent.com/66854273/236056648-e3f9ecc0-96fb-4641-a380-4633f2555691.png)

### What is the expected behavior?

An error is not thown.

### What actually happened?

An error is thrown.

### Suggested Solution/Workaround

The below error is what is thrown with /edebug on. Not sure of a workaround.

### Errors

```shell
Message: ...ce/AddOns/ElvUI/Mainline/Modules/Skins/GuildBank.lua:35: attempt to index field 'GuildBankTransactionsScrollFrame' (a nil value)
Time: Wed May  3 16:45:04 2023
Count: 1
Stack: ...ce/AddOns/ElvUI/Mainline/Modules/Skins/GuildBank.lua:35: attempt to index field 'GuildBankTransactionsScrollFrame' (a nil value)
[string "=(tail call)"]: ?
[string "@Interface/AddOns/ElvUI/Mainline/Modules/Skins/GuildBank.lua"]:35: in function <...ce/AddOns/ElvUI/Mainline/Modules/Skins/GuildBank.lua:16>
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua"]:1856: in function `CallLoadedAddon'
[string "@Interface/AddOns/ElvUI/Core/Modules/Skins/Skins.lua"]:1797: in function `?'
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:109: in function <...ies/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:19: in function <...ies/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceEvent-3.0/AceEvent-3.0.lua"]:120: in function <...UI_Libraries/Core/Ace3/AceEvent-3.0/AceEvent-3.0.lua:119>
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface/FrameXML/UIParent.lua"]:545: in function `UIParentLoadAddOn'
[string "@Interface/FrameXML/UIParent.lua"]:608: in function `loadFunc'
[string "@Interface/FrameXML/PlayerInteractionFrameManager.lua"]:179: in function `ShowFrame'
[string "@Interface/FrameXML/PlayerInteractionFrameManager.lua"]:215: in function <Interface/FrameXML/PlayerInteractionFrameManager.lua:212>

Locals: (*temporary) = <function> defined =[C]:-1

```
```


### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://www.tukui.org/download.php?ui=elvui&changelog).
- [X] I have verfied ElvUI is up to date by checking /estatus.
- [X] I have verfied my issue doesn't exist in the ElvUI Issue Tracker.
- [ ] I have not read step six and I am just clicking all boxes.
- [X] I have verfied my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the three called 'ElvUI', 'ElvUI Libraries', and 'ElvUI Options' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

**Comment by 1Botanica1** [comment]
This is fixed already in the dev build which you can download or wait until next release (no eta)

**Comment by jeremy-hill** [comment]
Thanks. Would you like me to close this?

**Comment by kodewdle** [comment]
fixed in 13.31
